### ISSUE TITLE
PA VERIF - add some verbiage on the dialog warning users not to share FTI from SSA

### ISSUE THREAD

**Issue by bblomer** [MN-Script-Team/DHS-MAXIS-Scripts#2103]
We received information during the Maxis Mentor call 2/24/16 from Brian Shields stating we are not allowed to verify information regarding income from other sources such as SSI/RSDI, VA benefits, income form a job.  However, this script pulls that information.  Our understanding is that this is from an FTI rule.  Can you clarify if we:
1. Can include that information?
2. Are supposed to take that information off before sending the letter 
3. Can it be corrected so it doesn’t pull that information?
Thanks,
Bridget Blomer
Scott County

**Comment by Farleight** [comment]
They did just recently clarify that any SSI info received from Social Security rather than from a client, is NOT able to be shared. I have not heard of this applying to other sources, but then with wages, etc, it almost always comes from the client in some form, or we don't technically have it "verified" anyway.

**Comment by theVKC** [comment]
Can I get some feedback from the other scriptwriters out there? @MN-Script-Team/county-agency-scriptwriters, any thoughts on this?

Also @ruthmaloney, if you can check on this when you get back, that'd be great. :) :)

**Comment by IlseFerris** [comment]
I agree with @Farleight, SSA information cannot be shared, nothing mentioned about other income from other sources were mentioned at mentor meeting

**Comment by courtrightd** [comment]
First, I'll take on modifying the script if we decide it's needed.  As for the policy end - this script was meant to be used when a signed release to share is provided, so I think @Farleight  is correct that we are good on anything provided by client.  I hesitate to remove it completely since all the Hud forms we receive have a line specifically for other income, and the client has released us to share that info. 
 A Possible solution would be to rewrite the logic so the script does not write SSI/RSDI into the other income field.
Other solution remove all amounts from the other income field, so it simply states income types known to the agency, and providers can request info directly.

**Comment by C-Love** [comment]
Here is my understanding:
1.True - we cannot verify for other agencies information that was received from a source other than MAXIS programs. Such as SSI/Jobs
2. When the script edit box comes up this information can be changed before the word document is written. So amounts could be removed but job name could be left in. So this could just be deleted by the script user.
3. If I recall correctly, this was left in as this document may be given directly to the client. We can certainly tell the client how we are budgeting income to determine benefits. 

Basically users need to determine the correct way to use these scripts within policy. It could be that clarification should be added to the instructions or to add some notes to the dialog to alert workers to check this clearly.

**Comment by Farleight** [comment]
This is a bit more complicated, but you could pull from the Verif Source on UNEA, and use the code to determine whether or not we can include that info. That would obviously require the panels to be coded correctly, which is probably asking too much.

**Comment by Y-Lor-Y** [comment]
My two cents:
1) SSI/RSDI amount that are provided from the client: documented in UNEA panel and case noted - these are not consider FTI information - as @Farleight  mentioned above.

2) Income - maybe I've skipped this from the updated PDF that DHS just sent out back beginning of this year 2016.  But, I've not heard any changes on income.

3) The script itself mainly and mostly allow the script user to input these information in.  I think that to refer or caution user of the FTI rules and to check with their supervisors/resource person on the dialog box, if they have question should do it.  User/Worker should only be verifying information that is 'issued' by the county or part of the county (like child support).  Any other sources, I would not even mention it.  The other agencies is really only asking for what we've given/issued to the client - even with the authorization for release information in place.  It is the client responsibility to provide employer or SSA information to that agency that is requesting for the county to verify the programs or other income that is under our control.

4) @courtrightd  I'm thinking if it'd also make sense to include the child supports disregard as an option on there, too?  Why I'm asking is because the MFIP Housing is on there, too.  

(not sure if drag/drop the item will show up on this - hopefully it does).  Just sharing the updated PDF of the FTI from DHS below.

[Guide to Safeguarding FTI & SSA Information DHS-285574.pdf](https://github.com/MN-Script-Team/DHS-MAXIS-Scripts/files/213909/Guide.to.Safeguarding.FTI.SSA.Information.DHS-285574.pdf)
[FTI  and SSA Protecting Information Document.docx](https://github.com/MN-Script-Team/DHS-MAXIS-Scripts/files/213910/FTI.and.SSA.Protecting.Information.Document.docx)

**Comment by bblomer** [comment]
Thanks Yang.  Now that I have re-read the guide it’s clear to me what we can release and what we can’t.  I think the script is fine and we can just take off information if it’s FTI…

The modifications are you suggesting look good too but I’ll leave that up to the group.

Thanks,

Bridget Blomer
Financial Assistance Supervisor
Scott County Health & Human Services
792 Canterbury Rd So
Shakopee, MN  55379
Phone: 952-496-8258
Fax: 952-496-8825


Privileged/Confidential information may be contained in this message.  If you are not the addressee indicated in this message (or responsible for delivery of the message to such person), please do not copy or deliver this message to anyone.  Rather, we ask that you: (1) please notify the sender by replying to this transmission and (2) please delete this message and any copy of it (in any form) without disclosing it.

**Comment by theVKC** [comment]
Anyone want to make changes to this script? If not, I'll close the issue.

**Comment by courtrightd** [comment]
I think I will add some verbiage on the dialog warning users not to share FTI from SSA.

**Comment by theVKC** [comment]
Think you can have it done by Friday for the release?

**Comment by courtrightd** [comment]
Yes, should be quick and easy.  I’ll try for tomorrow AM
